lighthouse-core/lib/network-recorder.js

```diff
@@ -333,7 +333,7 @@ class NetworkRecorder extends EventEmitter {
    * @return {NetworkRequest|null}
    * @private
    */
-  static _chooseInitiator(record, recordsByURL) {
+  static _chooseInitiatorRequest(record, recordsByURL) {
```

Comment:
Just naming changes in this file. `initiator` vs `initiatorRequest` tripped me up a little, and I found this small discrepancy.